src/simple_cache_client.rs

```diff
@@ -384,28 +425,28 @@ impl SimpleCacheClient {
     /// # Arguments
     ///
     /// * `cache_name` - name of cache
-    /// * `cache_key`
-    /// * `cache_body`
-    /// * `ttl_seconds` - If None is passed, uses the client's default ttl
+    /// * `cache_key` - key of entry within the cache.
+    /// * `cache_body` - data stored within the cache entry.
+    /// * `policy` - TTL policy to use. Note that set always sets the item TTL so refresh is ignored.
```

Comment:
The rename to `CollectionTtl` will make it more obvious, but this parameter should just be `Duration` to match our other sdks: https://github.com/momentohq/client-sdk-dotnet/blob/main/src/Momento.Sdk/ISimpleCacheClient.cs#L130